sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java

```diff
@@ -317,9 +319,21 @@ protected void addFields() {
 					FacadeProvider.getFacilityFacade()
 						.getActiveFacilitiesByDistrictAndType(districtDto, (FacilityType) facilityType.getValue(), true, false));
 			}
-			FieldHelper.updateItems(
-				cbPointOfEntry,
-				districtDto != null ? FacadeProvider.getPointOfEntryFacade().getAllActiveByDistrict(districtDto.getUuid(), true) : null);
+			// show all possible POEs for jurisdiction-district and placeofstay-district
+			if (differentPlaceOfStayJurisdiction.getValue()) {
```

Comment:
If `differentPlaceOfStay` is checked then it should show only the POE-s of the place of stay district to be consistent with the health facility selection.

If it needs to be filtered by both districts, then it should be a topic for a new change request and both POE and facility selection should be adopted and also the validation on the backend side.

Reply:
I see, I implemented it this way because of @bernardsilenou comment in the related ticket:
> if "place of stay of this case differs from responsible jurisdiction == True", then suggest POE in both regions in the dropdown for point of entry

Should we wait for feedback from @bernardsilenou / @kwa20 ?

Reply:
I would to do this similarly to health facility to be consistent and if needed, @bernardsilenou / @kwa20  would create a change request. 

I don't really say the reason for having a POE linked to jurisdiction district when there is a different place of stay district. As I know the POE should be the place where the case is located so if it has different place of stay then the POE must be in the place of stay district and not on the jurisdiction district.

@MateStrysewske what do you think?

Reply:
@DavidBaldsiefen I agree with @leventegal-she. There should not be a need to filter for both regions/districts at the same time because if the POE is in another region/district, it would not make sense to show the POE's in the jurisdiction

Reply:
Alright, then I will filter the POEs only based on the placeofstay (if provided). Otherwise based on jurisdiction.